rules/static.go

```diff
@@ -49,6 +53,57 @@ func (r *DefaultRuleRenderer) acceptAlreadyAccepted() []Rule {
 	}
 }
 
+func (r *DefaultRuleRenderer) StaticFilterInputForwardCheckChain(ipVersion uint8) *Chain {
```

Comment:
Think you need a comment to explain what each block of rules is for since this s quite fiddly code!

Reply:
Done.